Makefile

```diff
@@ -193,20 +193,20 @@ dist/argo-windows-%.gz: dist/argo-windows-%
 	gzip --force --keep dist/argo-windows-$*.exe
 
 dist/argo-windows-%: server/static/files.go $(CLI_PKGS) go.sum
-	CGO_ENABLED=0 $(GOARGS) go build -v -gcflags '${GCFLAGS}' -ldflags '${LDFLAGS} -extldflags -static' -o $@.exe ./cmd/argo
+	CGO_ENABLED=1 $(GOARGS) go build -v -gcflags '${GCFLAGS}' -ldflags '${LDFLAGS} -extldflags -static' -o $@.exe ./cmd/argo
```

Comment:
Seconding this, would definitely prefer to avoid `CGO` usage, especially as this will likely be released as a patch to 3.5.x. Wouldn't want to add anything that is potentially breaking

Reply:
I reverted this back since now it is cgo-free (https://github.com/argoproj/argo-workflows/pull/12736#discussion_r1519971792)